Sources/Commands/SwiftRunTool.swift

```diff
@@ -158,6 +166,9 @@ public struct SwiftRunTool: SwiftCommand {
                     try ProcessEnv.chdir(swiftTool.originalWorkingDirectory)
                 }
 
+                // SwiftTool can't handle signals anymore, so reset the signal handler to SIG_DFL before call exec()
+                signal(SIGINT, SIG_DFL)
```

Comment:
I agree with you. Should I create a wrapper just for `SwiftRunTool`? Or should I create it so that it can be used from other commands? If there is a possibility of using `exec` for other commands in the future, I feel that a wrapper should be created for the entire `Commands` target.

Reply:
good point. lets add to SwiftTool and then we can call that from any other tool

Reply:
Thank you for the direction. I will try.